courses/views/v2/__init__.py

```diff
@@ -598,3 +601,39 @@ def destroy(self, request, pk=None):
         )
 
         return self.list(request)
+
+
+@extend_schema(
+    description="Returns the json for the verifiable credential with the given ID, if it belongs to the user.",
+    responses={200: VerifiableCredentialSerializer(many=True)},
+)
+@api_view(["GET"])
+@permission_classes([AllowAny])
+def download_course_credential(request, credential_id):  # noqa: ARG001
+    credential = get_object_or_404(
+        VerifiableCredential,
+        pk=credential_id,
+    )
```

Comment:
It might be worth a comment that right now the Certificate and Credential UUID are the same (and it [seems like that'll be the case, at least for the immediate future](https://github.com/mitodl/mitxonline/pull/3147#discussion_r2629293528)) which is why we can pipe it directly in as the primary key.